xoa_driver/functions/anlt.py

```diff
@@ -247,217 +314,206 @@ async def an_log(port: GenericAnyPort) -> str:
-    :return:
-    :rtype: None
-    """
-    conn, mid, pid = port._conn, port.kind.module_id, port.kind.port_id
-    page_xindex = 8766
-    register_xindex = ((0xFFFF & lane) << 16) + 0x0002
-    rw = commands.PX_RW(conn, mid, pid, page_xindex, register_xindex)
-    await rw.set("0x00000000")
-    return None
-
-
-async def lt_nop(port: GenericAnyPort, lane: int) -> List[Token]:
-    """No operation for the lane, used to indicate interactive use
+async def lt_coeff_inc(port: GenericAnyPort, lane: int, emphasis: str) -> None:
+    """Ask the remote port to increase coeff of the specified lane.
 
     :param port: the port to configure
     :type port: :class:`~xoa_driver.ports.GenericAnyPort`
     :param lane: lane index, starting from 0
     :type lane: int
+    :param emphasis: coefficient index (-3, -2, -1, 0, 1)
+    :type emphasis: str
     :return:
     :rtype: None
     """
+    assert emphasis.lower() in ("pre3", "pre2", "pre", "main", "post"), "Para 'emphasis' not in (pre3, pre2, pre, main, post, PRE3, PRE2, PRE, MAIN, POST)!"
     conn, mid, pid = port._conn, port.kind.module_id, port.kind.port_id
-    page_xindex = 8766
-    register_xindex = ((0xFFFF & lane) << 16) + 0x0000
-    rw = commands.PX_RW(conn, mid, pid, page_xindex, register_xindex)
-    await rw.set("0x00010000")
+    
+    await apply(
+        commands.PL1_LINKTRAIN_CMD(conn, mid, pid, lane).set(cmd=LinkTrainCmd.CMD_INC, arg=LinkTrainCoeffs.from_str(emphasis))
+        )
     return None
 
 
-async def lt_coeff_inc(port: GenericAnyPort, lane: int, coeff: int, value: int) -> None:
-    """Increase coeff for a lane on a port
+async def lt_coeff_dec(port: GenericAnyPort, lane: int, emphasis: str) -> None:
+    """Ask the remote port to decrease coeff of the specified lane.
 
     :param port: the port to configure
     :type port: :class:`~xoa_driver.ports.GenericAnyPort`
     :param lane: lane index, starting from 0
     :type lane: int
     :param coeff: coefficient index (-3, -2, -1, 0, 1)
     :type coeff: int
-    :param value: the increase value
+    :param value: the decrease value
     :type count: int
     :return:
     :rtype: None
     """
-    assert coeff in range(-3, 1), "Para 'coeff' not in (-3, -2, -1, 0, 1)!"
+    assert emphasis.lower() in ("pre3", "pre2", "pre", "main", "post"), "Para 'emphasis' not in (pre3, pre2, pre, main, post, PRE3, PRE2, PRE, MAIN, POST)!"
     conn, mid, pid = port._conn, port.kind.module_id, port.kind.port_id
-    page_xindex = 8766
-    register_xindex = ((0xFFFF & lane) << 16) + 0x0000
-    aaaa = hex(value & 0xFFFF).replace("0x", "").zfill(4)
-    ce = {-3: 4, -2: 3, -1: 0, 0: 1, 1: 2}[coeff]
-    cc = hex(ce & 0xFF).replace("0x", "").zfill(2)
-    rw = commands.PX_RW(conn, mid, pid, page_xindex, register_xindex)
-    await rw.set(f"0x{aaaa}01{cc}")
+    
+    await apply(
+        commands.PL1_LINKTRAIN_CMD(conn, mid, pid, lane).set(cmd=LinkTrainCmd.CMD_INC, arg=LinkTrainCoeffs.from_str(emphasis))
+        )
```

Comment:
What for apply method here if here is only one single command which can be awaited on its own